lib/spot-checker.js

```diff
@@ -75,7 +75,16 @@ export default class SpotChecker {
       protocol: provider.protocol,
       address: provider.address,
       cid: task.cid,
-      stats,
+      dagScope: 'block',
+      stats: stats.blockRetrieval
+    })
+
+    await this.fetchCAR({
+      protocol: provider.protocol,
+      address: provider.address,
+      cid: task.cid,
+      dagScope: 'all',
+      stats: stats.fullRetrieval,
       maxByteLength
```

Comment:
Let me double check: I though we wanted to apply maxByteLength for block retrievals only? If we apply it for full retrievals, and use the 200 MB limit from Spark Checker (if I remember the limit correctly), then we are never going to download the entire file.

Could you please clarify?

Reply:
I imagined us to use this parameter to see which storage providers are only serving block retrievals and deny full file retrievals, without the need to download full file.